src/ckeditor/package.json

```diff
@@ -1,16 +1,17 @@
 {
 	"name": "ckeditor4-angular",
-	"version": "0.1.0",
-	"description": "Official Angular component for CKEditor 4.",
+	"version": "1.0.0",
```

Comment:
This version is bumped by publish script, but from what I see only in `dist/` folder. So we may bump it here too :+1: